tests/integration_tests/config/print_array_elements_utm_to_corsika.yml

```diff
@@ -0,0 +1,14 @@
+CTA_SIMPIPE:
+  APPLICATION: print_array_elements
+  CONFIGURATION:
+    INPUT: ./tests/resources/telescope_positions-North-utm.ecsv
+    EXPORT: corsika
+    USE_CORSIKA_TELESCOPE_HEIGHT: True
```

Comment:
my only worry with this approach is that the variables names are written in uppercase here, while in the applications they are lower case. That means, we would have to remember to always use the `-i` option from `grep` such that the search is case insensitive. See below
https://stackoverflow.com/questions/48492422/how-to-grep-for-case-insensitive-string-in-a-file

Reply:
I agree it is better to follow CTAO but I think it would be a good idea to include into the developer guidelines some info about this new way of testing functions, including also what I pointed out above. Of course that can be done later in another PR

Reply:
As mentioned in the call, it doesn't actually improve readability. 

Regarding following how CTAO generally writes yaml files, I am not sure the capitalisation is well motivated or that it is actually planned to be implemented that way. At least the yaml files which control ctapipe processing are not defined this way (see here for example - https://gitlab.cta-observatory.org/cta-computing/dpps/datapipe/pipeline-configurations/-/tree/main/ctapipe/v0.19.0/v3?ref_type=heads)

However, as I said in the call, my objection is limited to complaining, nothing more.